src/lib/components/Next/NextLinkLegacyBehavior.tsx

```diff
@@ -0,0 +1,15 @@
+import { useCallback, useState, type FC, type ReactElement } from 'react';
```

Comment:
This component seems to be like a "documented solution" not a flowbite-react component. So, maybe we could add it to the documentation insted? Some "Known Issues" section?